velox/functions/prestosql/aggregates/DecimalAggregate.h

```diff
@@ -208,6 +208,7 @@ class DecimalAggregate : public exec::Aggregate {
       });
     } else {
       rows.applyToSelected([&](vector_size_t i) {
+        clearNull(groups[i]);
```

Comment:
Shall we fix the case in line 193 and 236 as well?

Reply:
Updated.